controller/person.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  query,
+  update,
+  sparqlEscapeDateTime,
+  sparqlEscapeString,
+  sparqlEscapeUri,
+} from 'mu';
+import { v4 as uuid } from 'uuid';
+
+export async function getPersonByIdentifier(identifier: string) {
+  try {
+    const queryResult = await query(`
```

Comment:
this doesn't do anything that resources already does for us and we don't need it in this service to compute something else. would remove.